l10n_ec_ote/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
```

Comment:
This rules are also not required, as all models are declared on the dependencies.